handler/pingpong.go

```diff
@@ -0,0 +1,21 @@
+package handler
+import (
+	"../interfaces"
+	"strings"
+	"errors"
+)
+
+// Very simple ping-pong handler.
```

Comment:
comment on exported type PingPongHandler should be of the form "PingPongHandler ..." (with optional leading article)